doc/nrf/gs_programming.rst

```diff
@@ -87,4 +87,78 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
+
+1.    Open a terminal window.
+
+#.    Go to the samples or applications directory depending on whether you are building a sample or an application:
+
+      .. code-block:: console
+
+         cd nrf/samples
+         cd nrf/applications
+
+      .. note::
+         All the samples corresponding to the nRF9160 DK board are located at :file:`samples/nrf9160`.
+
+#.    Build the sample or application using the west command.
+      The development board is specified by the parameter ``board_name``, and the sample or application directory is specified by the parameter ``directory_name`` in the west command as follows:
+
+      .. code-block:: console
+
+         west build -b board_name directory_name
+
+      See `Board names <Board names_>`_ for more information on the development boards.
+      To reuse an existing build directory for building another sample or application for another board, pass ``-p=auto`` to ``west build``.
+
+      After running the ``west build`` command, the build files are generated in ``build/zephyr``.
+      For more information on the contents of the build directory, see `Zephyr documentation on Build Directory Contents`_.
+
+      .. include:: gs_programming.rst
+         :start-after: .. imp_note_nrf91_start
+         :end-before: .. imp_note_nrf91_end
+
+#.    Connect the development board to your PC using a USB cable.
+#.    Power on the development board.
+#.    Program the sample or application to the board from the build directory using the following command:
+
+      .. code-block:: console
+
+         west flash
+
+#.    Reset the board to start the sample or application.
+
+For more information on building and programming using command line, see the Zephyr documentation on :ref:`zephyr:west-build-flash-debug`.
+
+.. _gs_programming_board_names:
+
+Board names
+***********
+
+.. _table:
+
++-------------------+---------------------------------------------------+
+| Development Boards| Board names                                       |
++===================+===================================================+
+| nRF51-PCA10028    | nrf51_pca10028                                    |
++-------------------+---------------------------------------------------+
+| nRF52-PCA10040    | nrf52_pca10040                                    |
++-------------------+---------------------------------------------------+
+| nRF52840-PCA10056 | nrf52840_pca10056                                 |
++-------------------+---------------------------------------------------+
+| nRF5340           | nrf5340_dk_nrf5340_cpunet for network sample,     |
++                   +                                                   +
+|                   | nrf5340_dk_nrf5340_cpuapp for application sample  |
```

Comment:
```suggestion
|                   | nrf5340_dk_nrf5340_cpuapp (for the application sample)  |
```

Reply:
Removed the note on the samples/nrf9160.  Also removed the directory_name parameter and added information about this option in  a note.